risc0/zkvm/src/host/api/tests.rs

```diff
@@ -202,29 +201,29 @@ fn lift_join_identity() {
 
     let mut client = TestClient::new();
 
-    let session = client.execute(env, binary);
+    let session = client.execute(&env, binary);
     assert_eq!(session.segments.len(), client.segments.len());
 
     let opts = ProverOpts::default();
 
-    let receipt = client.prove_segment(opts.clone(), client.segments[0].clone());
-    let mut rollup = client.lift(opts.clone(), receipt);
+    let receipt = client.prove_segment(&opts, client.segments[0].clone());
+    let mut rollup = client.lift(&opts, &receipt);
 
     for segment in &client.segments[1..] {
-        let receipt = client.prove_segment(opts.clone(), segment.clone());
-        let rec_receipt = client.lift(opts.clone(), receipt);
+        let receipt = client.prove_segment(&opts, segment.clone());
+        let rec_receipt = client.lift(&opts, &receipt);
 
-        rollup = client.join(opts.clone(), rollup, rec_receipt);
+        rollup = client.join(&opts, &rollup, &rec_receipt);
         rollup
             .verify_integrity_with_context(&VerifierContext::default())
             .unwrap();
     }
-    let rollup_p254 = client.identity_p254(opts, rollup);
 
-    let rollup_receipt = Receipt::new(
-        InnerReceipt::Succinct(rollup_p254),
-        session.journal.bytes.into(),
-    );
+    // NOTE: identiy_p254 is generated, but never checked in this test.
+    // As long as the prover does not return an error, this test will pass.
```

Comment:
AFAIK, we don't have a way to run `identity_p254` verify in rust, because we don't have a circuit that supports this configuration.